### PR TITLE
add documentation to readme for setting up dev env

### PR DESCRIPTION
**Pull request by mockdeep**

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -114,6 +115,66 @@ installation and configuration. Information about reporting problems and
...
+config.ssh.forward_agent = true
+</pre>
+
+You'll need to run `vagrant reload` for the settings to take effect.
+
+h4. 2) log in to the vagrant box and clone RMagick
+
+<pre>
+vagrant ssh
+git clone https://github.com/gemhome/rmagick.git # or your fork
```
Comment (bf4) [inline README.textile]: I prefer to clone it, edit it, and commit/push changes outside of the ssh session.  I only use the ssh session to run tests (and install other rubies, packages etc).  If you commit within the session, I don't think it forwards your git config.  Try `git config --global --list | grep user`

**Comment by bf4** [comment]
:100: Awesome, thanks for sharing.  Now we just need to figure out and document cross-compilation :)

Sounds like we can Close with https://github.com/gemhome/rmagick/issues/163#issuecomment-72291944

File: README.textile (hunk 1 above)
Comment (mockdeep) [inline README.textile]: @bf4 Ah, okay. When I ran `vagrant ssh` it put me in the home directory and when I didn't see the project I assumed I didn't have access to it. Should have read the docs telling that it's in `/vagrant`.

**Comment by mockdeep** [comment]
@bf4 readme updated.

**Comment by bf4** [comment]
Looks good to me.   We'll need to add more info on building for release, but that can be in another PR

**Comment by mockdeep** [comment]
Yeah, I don't think I'm ready to dig into that part just yet. I think know that I've got a passing build I can start working on porting the tests over to rspec.

**Comment by bf4** [comment]
I'm ok to merge this.

**Comment by mockdeep** [comment]
Have at it. I don't have commit rights.

**Comment by bf4** [comment]
Ok, sent you an invite

**Comment by mockdeep** [comment]
Sweet, thanks!

**Pull request merged**